packages/hash/integration/src/tests/db/graph.test.ts

```diff
@@ -0,0 +1,170 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { DbClient, GraphClient } from "@hashintel/hash-api/src/db";
+import { getRequiredEnv } from "@hashintel/hash-backend-utils/environment";
+import { DataType, LinkType, PropertyType } from "@hashintel/hash-graph-client";
+
+const logger = new Logger({
+  mode: "dev",
+  level: "debug",
+  serviceName: "integration-tests",
+});
+
+const db: DbClient = new GraphClient(
+  { basePath: getRequiredEnv("HASH_GRAPH_API_BASE_URL") },
+  logger,
+);
+
+const accountId = { accountId: "00000000-0000-0000-0000-000000000000" };
+
+const textDataType$id = "https://example.com/data-type/v/1";
```

Comment:
Is there a way we can reuse some of the demo data in the graph package perhaps?

Reply:
We should definitely consider that. I think we can even import the JSON directly (but we'd have to reach into the directory by relative path).
I would like this to be something we consider for this [task](https://app.asana.com/0/1201095311341924/1202719361409869/f) (_internal_). I added a note about that :)